roles/ceph-nfs/defaults/main.yml

```diff
@@ -49,6 +49,12 @@ ceph_nfs_rados_export_index: "ganesha-export-index"
 # this bug: https://github.com/nfs-ganesha/nfs-ganesha/issues/217)
 # ceph_nfs_bind_addr: 0.0.0.0
 
+# Minimize or turn off ganesha's caching such as dentry and fd caching.
+# Currently, the caching is turned on by default.
+# When using ganesha as CephFS's gateway, it's recommended to turn off
+# ganesha's caching as the libcephfs clients also cache the same information.
+ceph_nfs_caching_off: false
+
```

Comment:
From the code it appears that `ceph_nfs_caching_off` is ignored when `ceph_nfs_file` is true. (I think this is the right thing.) The comment doesn't make that clear though.

Reply:
I think we can improve the option name: `ceph_nfs_cache`.

Reply:
> From the code it appears that ceph_nfs_caching_off is ignored when ceph_nfs_file is true. (I think this is the right thing.) The comment doesn't make that clear though.

OK. Added more info in the comments.

Reply:
> I think we can improve the option name: ceph_nfs_cache.

OK, changed to 'ceph_nfs_caching'. So if 'ceph_nfs_caching = true' , then doesn't it imply that ceph nfs caching is turned on by ceph-ansible by setting config options, which ceph-ansible doesn't. What if ganesha project decides to turn off caching by default, then 'ceph_nfs_caching = true' should set options to turn on caching?

I named is as 'ceph_nfs_caching_off', to be more explicit about what the option does, ceph-ansible only tries to disable caching and does not try to turn on caching. I think I should've named it 'ceph_nfs_disable_caching' instead.
@batrick what do you think?